tests/robot-tests/tests/admin_and_public/bau/prelease_visibility.robot

```diff
@@ -0,0 +1,223 @@
+*** Settings ***
+Resource  ../../libs/admin-common.robot
+Library  ../../libs/admin_api.py
+
+Force Tags  Admin  Local  Dev  AltersData
+
+Suite Setup       user signs in as bau1
+Suite Teardown    user closes the browser
+
+*** Variables ***
+${RELEASE_NAME}  Academic Year Q1 2020/21
+${RELEASE_NAME_2}  Academic Year Q1 2020/21 (not Live)
```

Comment:
I wouldn't have a separate variable here. But use `${RELEASE_NAME} (not Live)` in the place it is used.